editors/code/src/extension.ts

```diff
@@ -11,6 +11,13 @@ export function activate(context: vscode.ExtensionContext) {
 
 	ctx = new Ctx(context, lsp);
 	registerCommands(ctx);
+
+	return {
```

Comment:
This is a way to allow other extensions to call an API provided by Air. In this case we are exporting a private API for our own unit tests so they can access state in the Air instance.